vignettes/creating-custom-modules.Rmd

```diff
@@ -55,18 +57,20 @@ including an encoding panel on the left and main output covering the rest of the
 This function contains the shiny server logic for the module and should be of the form:
 
 ```
-function(id, datasets, ...) {
+function(id,
+         data, # optional, use if you want your module to access the application data
+         filter_panel_api, # optional, use if you want access to the filter panel from your module see teal.slice 
```

Comment:
This should say see "using the filter panel api in your modules" and that vignette should be created